src/schema/search.ts

```diff
@@ -201,9 +224,21 @@ export const resolvers: IResolvers<unknown, Context> = traceResolvers({
       getSession(ctx.userId, id),
     searchPostSuggestions: async (
       source,
-      { query }: { query: string },
+      { query, version }: { query: string; version: number },
       ctx,
     ): Promise<GQLSearchPostSuggestionsResults> => {
+      if (version === 2) {
+        const hits = await searchMeili(
+          `q=${query}&attributesToRetrieve=post_id,title`,
```

Comment:
It's fine for now but pretty sure they have something dedicated for auto complete

Reply:
Yeah they do, not sure if we just want to try it for now and see what we want to use?
They have a million and one options basically 😅

Just keeping it lean to evaluate for now (seems a tad slow still)

Reply:
No worries let's get it started and optimize later